distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1384,6 +1384,8 @@ Monitor system logs with the *LogViewer*, a convenient "viewing portal" for inco
 
 The LogViewer displays the most recent 500 log messages by default, but will grow to a maximum of 5000 messages.
 
+The top left button labeled POLLING/PAUSED will toggle live retrieval of incoming logs from the backend.
```

Comment:
Ignore my comment about the side-effects, just saw it's already described below.